exporters/memory/include/opentelemetry/exporters/memory/in_memory_span_exporter.h

```diff
@@ -0,0 +1,62 @@
+#pragma once
+#include "opentelemetry/sdk/trace/exporter.h"
+
+// Import for CMake
+// #include "in_memory_span_data.h"
```

Comment:
This should work for both CMake and Bazel.

https://gcc.gnu.org/onlinedocs/cpp/Search-Path.html

Reply:
The changes I made when I moved to a header-only implementation fixed that import issue, I just forgot to remove the comments. I've went ahead and fixed that